src/web/mjs/connectors/TenshiMoe.mjs

```diff
@@ -0,0 +1,154 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class TenshiMoe extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'tenshimoe';
+        super.label = 'Tenshi.moe';
+        this.tags = [ 'anime', 'english'];
+        this.url = 'https://tenshi.moe';
+        this.config = {
+            format: {
+                label: 'Language Settings',
+                description: 'Choose between Subbed or Dubbed anime. If your choice is not available the other is used as fallback',
+                input: 'select',
+                options: [
+                    { value: 'SUB', name: 'Subbed' },
+                    { value: 'DUB', name: 'Dubbed' }
+                ],
+                value: ''
```

Comment:
Shouldn't the value be one of the available options?

Reply:
I merely did like other connectors and It seems this doesnt pose a problem as i make sure to return something you are right.